### DISCUSSION TITLE
作者大大有将软件适配到安卓8.1的打算吗

### DISCUSSION THREAD

**Discussion by starrynight168** [LSPosed/LSPatch#222] [General]
🥺

**Comment by mario6714** [comment]
Maybe this help you:

https://github.com/WindySha/Xpatch

**Comment by xiaolianlang1** [comment]
8.1不是早就适配了吗

**Reply by mario6714** [reply]
I think no:

Supported Versions

    Min: Android 9
    Max: In theory, same with [LSPosed](https://github.com/LSPosed/LSPosed#supported-versions)
    
    Can i ask for 8.0?
